docs-ui/stories/overview.stories.mdx

```diff
@@ -1,6 +1,6 @@
-import StylesVariantsComponets from '../../images/overview-styles-variants-components.svg';
+import StylesVariantsComponets from '../images/overview-styles-variants-components.svg';
```

Comment:
I assume this image is specific to overview? I would structure it so that `overview` is its own directory and stick the image there too.